restapi/pom.xml

```diff
@@ -253,6 +247,7 @@
                             com.codahale.metrics,
                             com.codahale.metrics.*,
                             io.micrometer.core.*,
+                            !com.datastax.oss.driver.shaded.guava.*,
```

Comment:
We do need the driver code, but nothing exports shaded guava, and hence this is needed (OSGi error otherwise). Unfortunate that Guava shade is under some package as main driver.

Reply:
Note: was removed by a further commit.